client/ionic/src/pages/About.tsx

```diff
@@ -11,111 +12,83 @@ import {
-                  <b>Maintain distance between yourself and other people</b> if
-                  COVID-19 is spreading in your community. This is especially
-                  important for people who are at a higher risk of getting very
-                  sick, including the elderly.
-                </IonCardSubtitle>
-              </IonCardHeader>
-              <IonCardContent>
-                <IonIcon size="large" icon={peopleOutline}></IonIcon>
-              </IonCardContent>
-              <IonCardContent>
-                <IonButton href="/menu" shape="round">
-                  Learn More
-                </IonButton>
-              </IonCardContent>
-            </IonCard>
-          </IonSlide>
-        </IonSlides>
+        {flow.content && flow.content.screens && (
+          <IonSlides pager={true}>
+            {flow.content.screens.map(screen => {
+              switch (screen.type) {
+                case 'TextImage':
+                  return (
+                    <IonSlide>
+                      <IonCard>
+                        {screen.topImageUri && (
+                          /* TODO: actual css */ <IonCardContent>
+                            <IonImg
+                              style={{
+                                width: 300,
+                                marginLeft: 'auto',
+                                marginRight: 'auto',
+                              }}
+                              src={flow.imgPrefix + '/' + screen.topImageUri}
+                            />
+                          </IonCardContent>
+                        )}
+                        <IonCardHeader>
+                          <IonCardTitle>{screen.headingText}</IonCardTitle>
+                          {screen.bodyTexts &&
+                            screen.bodyTexts.map(txt => (
+                              <IonCardSubtitle>{txt}</IonCardSubtitle>
+                            ))}
+                        </IonCardHeader>
+                        {screen.bottomImageUri && (
+                          /* TODO: actual css */ <IonCardContent>
+                            <IonImg
+                              style={{
+                                width: 100,
+                                marginLeft: 'auto',
+                                marginRight: 'auto',
+                              }}
+                              src={flow.imgPrefix + '/' + screen.bottomImageUri}
+                            />
+                          </IonCardContent>
+                        )}
+                        <IonCardContent>
+                          <IonButton href="/menu" shape="round">
+                            Learn More
+                          </IonButton>
+                        </IonCardContent>
+                      </IonCard>
+                    </IonSlide>
+                  );
+                default:
+                /** TODO: Handle errors of unsupported screen types correctly. */
```

Comment:
note that this is a huge thing to consider when we think about versioning - users with older versions of an app will not be able to render new screen types. this definitely bears some larger thought, and possibly making our screen types more generic where possible

Reply:
Yes.  We may be able to use just 2 or 3 screen types with lots of optional elements, laid out in a responsive fashion.
We have versioning built into the flow urls.  When we make truly breaking changes we bump the flow endpoint version and consciously decide what content pre-break vs post-break users would receive.